src/Serilog.Sinks.Oracle/Serilog.Sinks.Oracle.UnitTests/test.runsettings

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>  
+<RunSettings>  
+  <!-- Configurations that affect the Test Framework -->  
+  <RunConfiguration>  
+    <MaxCpuCount>1</MaxCpuCount>  
+    <!-- Path relative to solution directory -->  
+    <ResultsDirectory>.\TestResults</ResultsDirectory>  
+  
+    <!-- [x86] | x64    
+      - You can also change it from menu Test, Test Settings, Default Processor Architecture -->  
+    <TargetPlatform>x86</TargetPlatform>  
+  
+    <!-- Framework35 | [Framework40] | Framework45 -->  
+    <TargetFrameworkVersion>FrameworkCore10</TargetFrameworkVersion>  
+  
+    <!-- Path to Test Adapters -->  
+    <!-- <TestAdaptersPaths>%SystemDrive%\Temp\foo;%SystemDrive%\Temp\bar</TestAdaptersPaths> --> 
+  </RunConfiguration>  
+  
+  <!-- Configurations for data collectors -->  
+  <DataCollectionRunSettings>  
+    <DataCollectors>  
+      <DataCollector friendlyName="Code Coverage" uri="datacollector://Microsoft/CodeCoverage/2.0" assemblyQualifiedName="Microsoft.VisualStudio.Coverage.DynamicCoverageDataCollector, Microsoft.VisualStudio.TraceCollector, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">  
+        <Configuration>  
+          <CodeCoverage>  
+            <ModulePaths>  
+              <Exclude>  
+                <ModulePath>.*CPPUnitTestFramework.*</ModulePath>  
+              </Exclude>  
+            </ModulePaths>  
+  
+            <!-- We recommend you do not change the following values: -->  
+            <UseVerifiableInstrumentation>True</UseVerifiableInstrumentation>  
+            <AllowLowIntegrityProcesses>True</AllowLowIntegrityProcesses>  
+            <CollectFromChildProcesses>True</CollectFromChildProcesses>  
+            <CollectAspDotNet>False</CollectAspDotNet>  
+  
+          </CodeCoverage>  
+        </Configuration>  
+      </DataCollector>  
+  
+    </DataCollectors>  
+  </DataCollectionRunSettings>  
+  
+  <!-- Parameters used by tests at runtime -->  
+  <TestRunParameters>  
+    <Parameter name="webAppUrl" value="http://localhost" />  
+    <Parameter name="webAppUserName" value="Admin" />  
+    <Parameter name="webAppPassword" value="Password" />  
+  </TestRunParameters>  
+  
+  <!-- Adapter Specific sections -->  
+  
+  <!-- MSTest adapter -->  
+  <MSTest>  
+    <MapInconclusiveToFailed>True</MapInconclusiveToFailed>  
+    <CaptureTraceOutput>false</CaptureTraceOutput>  
+    <DeleteDeploymentDirectoryAfterTestRunIsComplete>False</DeleteDeploymentDirectoryAfterTestRunIsComplete>  
+    <DeploymentEnabled>False</DeploymentEnabled>  
+    <AssemblyResolution>  
+      <Directory path="D:\myfolder\bin\" includeSubDirectories="false"/>  
```

Comment:
Could you explain this path here?
Is it necessary?

Reply:
Removed this file. Don't think we need this.